src/runtime/utils.ts

```diff
@@ -128,3 +124,53 @@ export function collectValues<F extends Record<string, RefObject<Field>>>(
         {} as Record<keyof F, unknown>,
     );
 }
+
+export function getSelectedAuth(prefix: string): {type: string | null; value: string | null} {
+    const type = sessionStorage.getItem(`${prefix}_type`);
+    return {
+        type,
+        value: sessionStorage.getItem(`${prefix}_value`),
+    };
+}
+export function getAuthByType(
+    prefix: string,
+    type: string,
+): {type: string | null; value: string | null} {
+    const typeFromStorage = sessionStorage.getItem(`${prefix}_type`);
+
+    return {
+        type: typeFromStorage,
+        value: type === typeFromStorage ? sessionStorage.getItem(`${prefix}_value`) : null,
+    };
+}
+
+export function setAuth(prefix: string, {type, value}: {type: string; value: string}) {
+    sessionStorage.setItem(`${prefix}_type`, type);
+    sessionStorage.setItem(`${prefix}_value`, value);
+}
+
+export function isV3SecurityApiKey(v3Security: V3Security): v3Security is V3SecurityApiKey {
+    return v3Security.type === 'apiKey';
+}
+
+export function isV3SecurityOAuth2(v3Security: V3Security): v3Security is V3SecurityOAuth2 {
+    return v3Security.type === 'oauth2';
+}
+
+export function isV3SecurityOAuthInline(
+    v3Security: V3Security,
+): v3Security is V3SecurityOAuthInline {
+    return (
+        isV3SecurityOAuth2(v3Security) &&
+        Boolean('x-inline' in v3Security && v3Security['x-inline'])
```

Comment:
Custom parameter for ignoring flows

Reply:
Wouldn't an option to switch between token acquisition flow and the inline mode be a better solution here? I don't think it's reasonable to assume that this flag will be adopted.

Reply:
No, inline is our specific case. It is important for us to keep the possibility of completely ignoring flows.